python/lsst/ip/isr/brighterFatterKernel.py

```diff
@@ -221,8 +223,10 @@ def fromDict(cls, dictionary):
         calib.shape = (dictionary['metadata'].get('KERNEL_DX', 0),
```

Comment:
The failure for the `_VERSION` increment can probably put in this method, as part of the `_OBSTYPE` check.  I think this must either have a case where `_VERSION == 1.0` is packed into the v1.1 format (with a warning), or a `RuntimeError` that the old version is unsupported.

Reply:
I think I'll produce a `RuntimeError`, as I don't see how I could turn, for example, `finalMeans` into `rawMeans`, and also create an `expIdMask` entry that might not correspond with reality (i.e., there's missing info. If we were going the other way, then it could be done... `rawMeans[expIdMask] = finalMeans`)

Reply:
I decided to make it backward compatible.